### PR TITLE
Setup typescript

### PR DESCRIPTION
**Pull request by hktouw**
@open-truss/engineers

### CODE REVIEW

File: packages/open-truss/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```
Comment (jonmagic) [inline packages/open-truss/tsconfig.json:3]: https://www.typescriptlang.org/tsconfig#target

```suggestion
    "target": "es6",
```

**Review by jonmagic — APPROVED** [review]
One suggestion. Where did you get the `tsconfig.json` from? Was it auto created? Just wondering if there is a best practices starting point for a tsconfig in a new project.

**Comment by hktouw** [comment]
Oh, I more or less copied the one @kmcq made in the demo app https://github.com/open-truss/open-truss/blob/main/demo-app/tsconfig.json

**Comment by kmcq** [comment]
> Oh, I more or less copied the one @kmcq made in the demo app https://github.com/open-truss/open-truss/blob/main/demo-app/tsconfig.json

Just to add the remaining context: this came from the create-next-app starter script

**Pull request merged**